### PR TITLE
[com_actionlog] Fix username log checkin actions

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request for PR #23526 

### Summary of Changes

Based on the function onUserAfterSave, the variable username was set with username and not user->name

### Testing Instructions

Install 3.9.3-rc
Go to User -> User Action Log

### Expected result

User **username** performed a check in to table #__content
username = Login Name

### Actual result

User **Super User** performed a check in to table #__content
**Super User** here is the name (Name of the user) and not the username.

### CODE REVIEW

File: plugins/actionlog/joomla/joomla.php [hunk 1]
```diff
@@ -924,25 +924,28 @@ public function onUserAfterRemind($user)
...
 		$context = $this->app->input->get('option');
-		$user    = Factory::getUser();
+		$jUser    = Factory::getUser();
 
 		if (!$this->checkLoggable($context))
 		{
 			return;
 		}
 
+        $userId   = $jUser->id ?: $user['id'];
```
Comment (Quy) [inline plugins/actionlog/joomla/joomla.php]:
```suggestion
		$userId   = $jUser->id ?: $user['id'];
```

File: plugins/actionlog/joomla/joomla.php [hunk 2]
```diff
@@ -924,25 +924,28 @@ public function onUserAfterRemind($user)
...
+        $username = $jUser->username ?: $user['username'];
```
Comment (Quy) [inline plugins/actionlog/joomla/joomla.php]:
```suggestion
		$username = $jUser->username ?: $user['username'];
```

File: plugins/actionlog/joomla/joomla.php [hunk 3]
```diff
@@ -924,25 +924,28 @@ public function onUserAfterRemind($user)
 	public function onAfterCheckin($table)
 	{
 		$context = $this->app->input->get('option');
-		$user    = Factory::getUser();
+		$jUser    = Factory::getUser();
```
Comment (Quy) [inline plugins/actionlog/joomla/joomla.php]:
```suggestion
		$jUser   = Factory::getUser();
```

**Comment by HLeithner** [comment]
@alikon can you check this fix for your PR?

**Comment by SharkyKZ** [comment]
This introduces warnings because `$user` is now undefined. Reverting these changes and just replacing `$user->name` with `$user->username` should be fine.

**Comment by alikon** [comment]
@HLeithner sure

@all see #23852

**Pull request closed without merging**